src/global/less/corporate-ui/modal.less

```diff
@@ -153,20 +151,24 @@
   
 }
 
-@media (max-width: 991px) {
+@media (max-width: 992px) {
```

Comment:
Are u sure that this is infact correct, I changed this a while back.

Reply:
Can you change to 991 for me? Screaming kids right now. Damn!